packages/canvas/canvas-extract/global.d.ts

```diff
@@ -0,0 +1,7 @@
+declare namespace GlobalMixins
+{
+    interface CanvasRenderer
+    {
+        extract: import('@pixi/canvas-extract').CanvasExtract;
```

Comment:
isn't this `plugins.extract`?

Reply:
yes, i didnt see that. Looks like i was too eager to use the new Global Mixins!

Reply:
fixed